src/components/v2/MyAccount/MyAccount.tsx

```diff
@@ -0,0 +1,106 @@
+/** @jsxImportSource @emotion/react */
+import React, { useState } from 'react';
+import Box from '@mui/material/Box';
+import Paper from '@mui/material/Paper';
+import { ISliderProps, Slider } from '../Slider';
+import { useMyAccountStyles as useStyles } from './MyAccountStyles';
+import { Toggle } from '../Toggle';
+import { Icon } from '../Icon';
+import { Tooltip } from '../Tooltip';
+
+const myAccountResources = {
+  myAccountTitle: 'My account',
+  apyWithXvs: 'APY with XVS',
+  apyWithXvsTooltip: 'tooltip content',
+  netApy: 'Net APY',
+  dailyEarnings: 'Daily earnings',
+  supplyBalance: 'Supply balance',
+  borrowBalance: 'Borrow balance',
+  borrowLimit: 'Borrow limit:',
+  safeLimit: 'Your safe limit:',
+};
+
+interface IMyAccountProps {
+  netApy: string;
+  dailyEarnings: string;
+  supplyBalance: string;
+  borrowBalance: string;
+  borrowLimit: string;
+  borrowLimitSum: string;
+  safeLimit: string;
+  sliderProps: ISliderProps;
+}
+
+export const MyAccount = ({
+  netApy,
+  dailyEarnings,
+  supplyBalance,
+  borrowBalance,
+  borrowLimit,
+  borrowLimitSum,
+  safeLimit,
+  sliderProps,
+}: IMyAccountProps) => {
+  const [isToggleSwitched, setToggleSwitched] = useState(true);
+  const handleSwitch = (event: React.ChangeEvent<HTMLInputElement>, checked: boolean) => {
+    setToggleSwitched(checked);
+  };
+  const styles = useStyles();
+  return (
+    <Box css={styles.root} component={Paper}>
+      <div css={styles.row}>
+        {myAccountResources.myAccountTitle}
+        <span css={styles.apyWithXvs}>
+          <Tooltip css={styles.tooltip} title={myAccountResources.apyWithXvsTooltip}>
+            <span>
```

Comment:
Just curious; what do you need the `span` there for?

Reply:
> Tooltip is not working properly because the child of a Material-UI Tooltip must be able to hold a ref.

https://stackoverflow.com/questions/57527896/material-ui-tooltip-doesnt-display-on-custom-component-despite-spreading-props


going to move wrapper for children directly into tooltip component

Reply:
I see! Thanks for the info :)